packages/lib/models/Setting.ts

```diff
@@ -1504,6 +1504,22 @@ class Setting extends BaseModel {
 				storage: SettingStorage.Database,
 			},
 
+			updatedDefaultPluginsInstallStates: {
+				value: false,
+				type: SettingItemType.Bool,
+				public: false,
+				appTypes: [AppType.Desktop],
+				storage: SettingStorage.File,
+			},
+
+			defaultPlugins: {
+				value: {},
+				type: SettingItemType.Object,
+				public: true,
+				appTypes: [AppType.Desktop],
+				storage: SettingStorage.Database,
+			},
```

Comment:
Any reason why this is a dynamic settings? Feels like it should just be a constant since it doesn't change for a given version?

Reply:
Not sure what you mean by Dynamic. Are you suggesting I put it in some `const` variable? Because I think by putting `defaultPlugins` in `Setting.ts` it will be easier to retrieve it from anywhere.

Reply:
I'd just put a defaultPlugins.ts file in lib/services/plugins. That way it can be easily included anywhere. Putting it in Setting.ts could cause dependency loops since this file is already included in many places.

Reply:
Hey, I was thinking of making this `defaultPlugins.ts` as a class which will contain all the plugins' path, IDs etc. Do you think I should go ahead with this?

Reply:
If it's a constant, just make it constant? Anything else I don't know